source/includes/steps-run-as-daemon.rst

```diff
@@ -0,0 +1,52 @@
+.. procedure::
+   :style: normal
+
+   .. step:: Start the daemon.
+
+      Run the following command to start the daemon:
+
+      .. code-block::
+
+         docker run -d --name mongodb/atlas mongodb/atlas
+
+   .. step:: Get a shell.
+
+      Run the following command to get a shell with an environment file:
+      
+      .. code-block:: 
+          
+         docker exec --env-file atlas.env --rm -it mongodb/atlas bash
+
+   .. step:: Authenticate and run {+atlas-cli+} commands.
+
+      To authenticate and run commands, follow the steps to :ref:`connect from the {+atlas-cli+}
+      <connect-atlas-cli>`, adding 
+      ``docker exec --env-file ./atlas.env --rm mongodb/atlas`` before
+      each {+atlas-cli+} command. For example, to run the 
+      :ref:`atlas config init <atlas-config-init>` command with an
+      environment variable file, run the following command, replacing
+      ``atlas.env`` with the name of the environment file:
+      
+      .. code-block::
+         
+         docker exec --env-file ./atlas.env --rm mongodb/atlas atlas config init
+
+      .. note::
+
+         The ``atlas config init`` command requires setting API keys in the `environment variable file 
```

Comment:
Atlas config init is used to set up the API keys (so it's manual authentification). I think this should be removed and just a note that says that you should add the API keys to your env file.

The idea is that if you are running as a daemon you will never authenticate manually (so interactively)

Reply:
Got it, removed those instructions and changed to "add API keys"